### PR TITLE
QPPSF-5436: Update CMS159v7 descriptions

### PR DESCRIPTION
**Pull request by saquino0827**
#### Motivation for change

Add an accurate description for both strata scenarios

#### Release checklist:
Tasks that must be done prior to merging this PR, including testing.

* [x] [Package version updated](https://github.com/CMSgov/qpp-measures-data/blob/master/CONTRIBUTING.md#versioning-publishing-and-creating-new-releases)
* [x] Documentation updated
* [x] Unit tests added/passing
* [x] Verified working locally

##### Associated JIRA tickets:
* https://jira.cms.gov/browse/QPPSF-5436

### CODE REVIEW

**Comment by saquino0827** [comment]
@harrisj Please review when you have the time.

**Comment by saquino0827** [comment]
@KyleApfel Thanks for the reminder. I forgot to push the updated package json version.

**Comment by jenjaina** [comment]
Any chance of getting this merged in soon? 🙏

**Comment by saquino0827** [comment]
@jenjaina Just waiting on a code owner to review but if @KyleApfel's a code owner I can. I'll message @harrisj and see if he can get someone to review.

File: measures/2019/measures-data.json [hunk 1]
```diff
@@ -30572,7 +30572,8 @@
         "name": "12-17"
       },
       {
-        "name": "18+"
+        "name": "18+",
+        "description": "Percentage of adult patients (aged 18 years or older) with a diagnosis of major depression or dysthymia and aninitial PHQ-9 or PHQ-9M score greater than nine during the index event who reached remission at twelve months as demonstrated by a twelve month (+/-60 days) PHQ-9 or PHQ-9M score of less than 5."
       }
```
[the remark this answers is not in the document]
Comment (saquino0827) [reply measures/2019/measures-data.json]: @shebryant yes, I did it progromattically using an rpm build on 2019.

**Comment by saquino0827** [comment]
@chouinar yes, I did to a rebase on master but I guess I didn't push it. Should be fixed now.

**Pull request merged**